SpeechResponder/VariablesWindow.xaml.cs

```diff
@@ -1,4 +1,4 @@
-﻿using EddiEvents;
+using EddiEvents;
```

Comment:
Not at my computer so not sure why this is showing up as a diff. Whitespace issue?

Reply:
Nothing changed here. It shouldn't be included. I don't see any whitespace differences but that it the most likely reason for it to appear here.